python/grass/temporal/abstract_space_time_dataset.py

```diff
@@ -1369,15 +1369,22 @@ def get_registered_maps_as_objects_with_gaps(
         In case more map information are needed, use the select()
         method for each listed object.
 
+        The combination of the spatial_extent and spatial_relation parameters
+        can be used to return only map objects with the given spatial relation
+        to the provided spatial extent
```

Comment:
```suggestion
        to the provided spatial extent.
```

And same below (3x).